newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -83,6 +84,11 @@ export default function SummaryPage() {
   const mailSent = sendMailResponse !== null;
   const editUrl = `/newdle/${newdle.code}/edit`;
 
+  const createEvent = async (event, {value}) => {
+    const resp = await _createEvent(newdle.code);
+    window.open(resp.url);
```

Comment:
This triggers the popup blocker. It also fails badly if `resp` is undefined due to an error. Easiest solution would be this:

```js
if (resp && resp.url) {
  location.href = resp.url
}
```

Reply:
This results in this error:
```
/home/adeel/software/newdle/newdle/client/src/components/summary/SummaryPage.js
  90:7  error  Unexpected use of 'location'  no-restricted-globals
```
Should I suppress it?

Reply:
yes, perfectly valid here...

Reply:
Thanks, had to use `window.location.href` otherwise got an error when committing.